public/achievementInfo.php

```diff
@@ -121,8 +121,21 @@ function generateAchievementMetaDescription(
 RenderContentStart($pageTitle);
 ?>
 <?php if ($permissions >= Permissions::Developer || ($permissions >= Permissions::JuniorDeveloper && $isAuthor)): ?>
+    <?php
+        $canHaveTypes = mb_strpos($gameTitle, "[Subset") === false && mb_strpos($gameTitle, "~Test Kit~") === false;
```

Comment:
Suggest extending this to Events

Reply:
Updated to check for games that have a console ID of 101.